restore-cognito.md

```diff
@@ -0,0 +1,32 @@
+Prerequisites 
+For creating and executing backups, you can use the npm package called `cognito-backup-restore` 
+(do `npm install cognito-backup-restore` to get the package)
+
+Required IAM permissions
+If you want to create a backup of the users you will need the permission - `ListUsers`
+Restoring the users requires - `AdminCreateUser`
+There might be more roles required to do all the operations for getting and restoring users. If you need a permission added
+speak with Iva or Pol, who will add it through IAM.
+
+**Downloading user data**
+Here do either 1 or 2 depending on the situation.
+
+1. In case there is no cognito user pool or the user pool is empty, we have weekly user backup which executes every sunday at 1am. 
```

Comment:
I would keep point 1 only to say `In case there is no cognito user pool or the user pool is empty.`. Then., in a new line under that point I will add the "we have weekly user backup which executes every sunday at 1am.", but I will change the text to just say "we have a weekly backup" -- that makes the runbook less likely to go out of date. For example, with the sentence as is right now, if we changed the backup to not be at 1am or not be on Sunday, the text will not be correct anymore.